### ISSUE TITLE
[Bug Report]Collapse v-model bug

### ISSUE THREAD

**Issue by abielzhang** [iview/iview#3467]
### Environment
Latest version of Vue And Chrome

### Reproduction link
[https://jsfiddle.net/jps4mdxo/](https://jsfiddle.net/jps4mdxo/)

### Steps to reproduce
When setting Panel data, set the v-model value of Collapse

### What is expected?
The value of v-model is valid

### What is actually happening?
The value of v-model is invalid

**Comment by 1862** [comment]
```
this.$nextTick(function () {
    this.value1 = '11';
});
```

**Comment by Abiel1024** [comment]
@1862 This can solve the problem, but it consumes more resources.

**Comment by 1862** [comment]
it should be , cause it push a function to microtask or task.

**Comment by David-Tsui** [comment]
@Abiel1024 No, it's just a standard api when dynamic setting some data which would associate with UI.

**Comment by Abiel1024** [comment]
@David-Tsui Maybe we should think of a better solution
